packages/connect/README.md

```diff
@@ -57,6 +72,8 @@ With regards to this repo - All updates should go to current version branch, the
 
 For deployment process of trezor-connect v8 refer to [trezor/connect repository](https://github.com/trezor/connect/blob/develop/docs/deployment/index.md)
 
+Version 8 will be deprecated by the end of 2023.
```

Comment:
it is deprecated now, isn't it? Didn't we want to say it will be killed? removed?

Reply:
cc @Hannsek